optimade/validator/validator.py

```diff
@@ -5,50 +5,47 @@
 """
 # pylint: disable=import-outside-toplevel
 
-import time
-import requests
+import re
 import sys
 import logging
+import random
 import urllib.parse
-import traceback as tb
+from typing import Union, Tuple, Any
 
 try:
     import simplejson as json
 except ImportError:
     import json
 
-from typing import Union
-
-from pydantic import ValidationError
+import requests
 from fastapi.testclient import TestClient
 
-from optimade.models import InfoResponse, EntryInfoResponse, IndexInfoResponse
+from optimade.models import InfoResponse, EntryInfoResponse, IndexInfoResponse, DataType
 
-from optimade.validator.data import MANDATORY_FILTER_EXAMPLES, OPTIONAL_FILTER_EXAMPLES
-from optimade.validator.validator_model_patches import (
+from .utils import (
     ValidatorLinksResponse,
     ValidatorEntryResponseOne,
     ValidatorEntryResponseMany,
     ValidatorReferenceResponseOne,
     ValidatorReferenceResponseMany,
     ValidatorStructureResponseOne,
     ValidatorStructureResponseMany,
+    Client,
+    test_case,
+    print_failure,
+    print_notify,
+    print_success,
+    print_warning,
+    ResponseError,
 )
 
 
 BASE_INFO_ENDPOINT = "info"
 LINKS_ENDPOINT = "links"
+VERSIONS_ENDPOINT = "versions"
+NON_ENTRY_ENDPOINTS = (BASE_INFO_ENDPOINT, LINKS_ENDPOINT, VERSIONS_ENDPOINT)
 REQUIRED_ENTRY_ENDPOINTS = ["references", "structures"]
```

Comment:
I just tried to keep anything that was hard-coded away from the rest so that we can unpick it at a later date. Of course, the `versions` endpoint name is already hard-coded in our server itself... Certainly a lot of the stuff up the top of this module can be moved to utils or removed.

Reply:
As I wanted to say here, maybe move it to `config.py` at some point? :)

Reply:
Ended up making a `BaseConfig` at `optimade.validataor.config`, though not sure it will ever be touched...